### PR TITLE
Adds draft system requirements to repo.

### PR DESCRIPTION
**Pull request by jlamiano**
Seeking feedback and suggestions for system requirements

### CODE REVIEW

**Review by plyons72 — CHANGES_REQUESTED** [review]
Super minor, but on line 10, "motions" should be "motion"

**Comment by jlamiano** [comment]
No such thing as too minor. Thanks for catching it. Checkout fix in commit: bba0baad96
@plyons72

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
 # Software
 The Embedded Software Repo
+
+
+## System Requirements
+
+1. The PowerPlant shall be capable of operating for greater than 24 hours on a single full charge of its battery
+2. The PowerPlant shall stabilize its location to within 50 cm of the local maximum of light intensity
```
Comment (lindseymarra) [inline README.md]: Recommendation is to change this to 10 cm

Comment (jlamiano) [reply README.md]: Good catch. 50 cm would have been way too big. Checkout commit: 5e97aff6a112cc7

**Review by plyons72 — APPROVED** [review]
Works for me. Thanks Josh!

**Review by lindseymarra — APPROVED** [review]

**Review by jrm203 — APPROVED** [review]

**Pull request merged**